sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/BoundingBox.java

```diff
@@ -24,13 +25,17 @@
      * @param points The list of coordinates of the Bounding box.
      */
     public BoundingBox(final List<Point> points) {
-        this.points = points;
+        if (points == null) {
+            this.points = null;
+        } else {
+            this.points = Collections.unmodifiableList(points);
+        }
     }
 
     /**
      * Gets the list of all point coordinates of the bounding box.
      *
-     * @return The list of all point coordinates of the Bounding box.
+     * @return The unmodifiable list of all point coordinates of the Bounding box.
```

Comment:
immutable?

Reply:
unmodifiable is more of a Java keyword to help understand what we do underneath.